src/Conference/ConferenceDetails.jsx

```diff
@@ -44,21 +53,38 @@ const ConferenceDetails = ({ conference, id, isLoading, error }) => {
     deleteConference(id);
   };
 
+  const expressInterest = () => {
+    addFavouriteConference(userId, conferenceId, token);
+  };
+
   return (
     <StyledForm>
-      <StyledCard>
+      <Card>
         <StyledCardHeading className="name">{name}</StyledCardHeading>
-        <StyledLink to={`/${id}/edit`}>Edit Conference</StyledLink>
+        {token && (
+          <StyledLink className="editLink" to={`/${id}/edit`}>
+            Edit Conference
+          </StyledLink>
+        )}
         <p className="topic">{topic}</p>
         <p className="date">{getDatestring(date)}</p>
         <p className="time">{getTimestring(date)}</p>
         <p className="city">{city}</p>
         <StyledDescription className="description">
           {description}
         </StyledDescription>
-        <Button onClick={deleteThisConference}>Delete Conference</Button>
-      </StyledCard>
+        {token && (
```

Comment:
Am I right in saying I could delete your conference? Is this expected?

Reply:
Yes, at the moment we've built it so a logged-in user can add, delete and express interest in any conference. We don't currently have the functionality to delete only your own conferences, but I can add in another ticket for this!